optimum/intel/openvino/configuration.py

```diff
@@ -68,10 +68,16 @@ def add_input_info(self, model_inputs: Dict):
             for name, value in model_inputs.items()
         ]
 
-    def _enable_standard_onnx_export_option(self):
-        if isinstance(self.compression, dict):
+    def _set_standard_onnx_export_option(self):
+        # This method depends on self.save_onnx_model.
+        # save_onnx_model is defaulted to false so that the final model output is
+        # in OpenVINO IR to realize performance benefit in OpenVINO runtime.
+        # True value of save_onnx_model will save a model in onnx format.
+        # Quantized performance of onnx model in onnxruntime will require OpenVINO backend.
+        # Otherwise, default onnxruntime backend gives a full-precision performance instead of low-precision's.
```

Comment:
I am concerned that referring people to ORT here will cause some confusion. This is a private method for people who need it, but the default supported method to run models created with optimum[openvino] is with OpenVINO Runtime.